### DISCUSSION TITLE
IP Combinability

### DISCUSSION THREAD

**Discussion by Spablob** [storyprotocol/protocol-core-v1#181] [General]
Two IPs can be remixed in 1-click whenever the license terms are compatible.

These IP combinability restrictions are also present in the real world though and being onchain doesn't exactly alleviate license terms compatibility issues - that's why we often see custom agreements with lawyers in the real world.

- We can indeed make a 1-click remix when license terms are compatible -> improvement from the real world
- But it will revert when license terms are incompatible -> same as in the real world

Questions relative to the second point become:
- How to lower the overall probability of license incompatibility?
- If license incompatibility happens what could be ways to achieve compatibility (at least in some cases) while minimizing the need for both parties to talk/negotiate?

**Comment by Ramarti** [comment]
We need to beef up our license compatibility to fully get that effect @Spablob , what you describe sounds "work as intended" to me.

Are we missing context?

The UX improvement could be to have a license bid module where you can place some money for a specific term config, requesting it to the rights holder

**Reply by Spablob** [reply]
bidding for a specific term config could be worth exploring

**Comment by kingster-will** [comment]
> Two IPs can be remixed in 1-click whenever the license terms are compatible.

Could you further clarify this? 
Does it mean a child IP can derivative from two parents IPs? if so, it is the current behavior of protocol v1 that can make a child derivative from multiple parents with one call.

**Reply by Spablob** [reply]
Updated the original message - what I was trying to allude to/initiate a discussion on was:
- on how to lower the probability of license incompatibility
- and if license incompatibility were happen what could be ways to achieve compatibility (at least in some cases) while minimizing the need for both parties to talk/negotiate
